pyproject.toml

```diff
@@ -72,3 +72,16 @@ exclude = [
 
 [tools.pytest.ini_options]
 testpaths = ["tests"]
+
+[tool.ruff]
+# Allow lines to be as long as 140 characters.
+line-length = 140
```

Comment:
Manually solved ~90 errors to set the line length to 140. Going down to 120 or 100 requires solving 103 or 853 more errors, respectively. I propose we keep this at 140 for now and update it once we have an automatic code formatting tool. We can still write all new code according to 100 chars rule. @waltsims wdyt?

Reply:
Sounds good. I've added it to the project.